ansible/roles/kubernetes-addons/templates/dns/kube-svc.yaml.j2

```diff
@@ -1,4 +1,4 @@
-# This file should be kept in sync with cluster/gce/coreos/kube-manifests/addons/dns/skydns-svc.yaml
+# This file should be kept in sync with cluster/gce/coreos/kube-manifests/addons/dns/kubedns-svc.yaml
```

Comment:
The problem is not inside the file. The problem is the filename itself. Do an `ls` in the `roles/kubernetes-addons/templates/dns/` directory to see the problem. You renamed `skydns-svc.yaml.j2` to `kube-svc.yaml.j2`. I believe we both agree it should have been called `kubedns-svc.yaml.j2`